internal/util/fileutil.go

```diff
@@ -2,11 +2,14 @@ package util
 
 import (
 	"errors"
+	"fmt"
 	"io"
 	"net/http"
 	"os"
 )
 
+var DEFAULT_BUFFER_SIZE int64 = 1000
```

Comment:
This can be moved to constants.go file

Reply:
Yes but I thought it can be more useful if util package to be used independently by the other project.
How do you think about it?

Reply:
Yeah of course it should.
I just mean that it is better to move DEFAULT_BUFFER_SIZE variable only to constants.go file.

Reply:
ok, I've changed it!